src/Sushi.Mediakiwi.Headless/ScriptTagHelperComponent.cs

```diff
@@ -19,6 +19,40 @@ public class ScriptTagHelperComponent : TagHelperComponent
         [ViewContext]
         public ViewContext ViewContext { get; set; }
 
+        private void PerformReplacements(TrackingScriptSimple script)
+        {
+            if (script?.ReplacementItems?.Count > 0)
+            {
+                // Are all marked as mandatory ?
+                var allMandatory = script.ReplacementItems.Count == script.ReplacementItems.Count(x => x.MustBeFilled);
+
+                // are all values filled ?
+                var allFilled = script.ReplacementItems.Count == script.ReplacementItems.Count(x => string.IsNullOrWhiteSpace(x.Value) == false);
```

Comment:
Use script.ReplacementItems.All(x => string.IsNullOrWhiteSpace(x.Value) == false) for better performance

Reply:
Agreed (though it should be All, not Any), done